wiki/Glossary/en.md

```diff
@@ -591,22 +578,22 @@ The offset is the first half of beatmap timing, where the other half is the BPM.
 
 osu! has three meanings:
 
--     the rhythm video game for the Microsoft Windows Operating Systems. Spelled with a lowercase "o" and an exclamation mark at the end.
--     osu!standard, the game mode that osu! was originally based on.
--     An informal greeting in the Japanese language, resulting from condensing the phrase "Ohayo gozaimasu!" (Good morning!) Pronounced with a silent u ("OSS"). Close English equivalents include "Yo!" or "Hey!". To read more about its meaning in Japanese language and culture, see [Does anyone here speak/learn Japanese?](http://osu.ppy.sh/forum/viewtopic.php?p=2686#p2686).
+- the rhythm video game for the Microsoft Windows Operating Systems. Spelled with a lowercase "o" and an exclamation mark at the end.
+- osu!standard, the game mode that osu! was originally based on.
+- An informal greeting in the Japanese language, resulting from condensing the phrase "Ohayo gozaimasu!" (Good morning!) Pronounced with a silent u ("OSS"). Close English equivalents include "Yo!" or "Hey!". To read more about its meaning in Japanese language and culture, see [Does anyone here speak/learn Japanese?](https://osu.ppy.sh/forum/viewtopic.php?p=2686#p2686).
```

Comment:
Change the forum link to its current one (`https://osu.ppy.sh/forum/p/2686`), mostly for cosmetic purposes